jkwest-93/Smell-Ya-Later
# Jkw ui overhaul

#### What's this PR do?
- Makes a number of UI changes to clean up app
- Installs `react-collapsible` to handle a drop down & `react-moment` to handle time
- Restyles the Header to add an icon & a dropdown/About section
- Restyles `Item` cards and `ActivityDetail` cards for a better aesthetic
#### Where should the reviewer start?
- The work in this PR is spread across a number of different files! Check them all out
#### How should this be manually tested?
- Run `npm start` and demo in the browser; ensure connection to the back end so that data displays correctly
#### Any background context you want to provide?
- n/a
#### What are the relevant tickets?
- Close #24 
#### Screenshots (if appropriate)
- ![jkw-UI-overhaul](https://user-images.githubusercontent.com/62263439/93720409-05189100-fb46-11ea-9033-30654b903243.gif)
#### Questions:
- n/a
